examples/image-filter.js

```diff
@@ -15,7 +15,7 @@ const imagery = new TileLayer({
     url: 'https://api.maptiler.com/maps/satellite/{z}/{x}/{y}.jpg?key=' + key,
     tileSize: 512,
     maxZoom: 20,
-    crossOrigin: '',
```

Comment:
Can be either `'anonymous'`, `'use-credentials'`

Reply:
Where an `XYZ` source has been changed to `ImageTile` a `crossOrigin` setting would have been required, but it now defaults to `'anonymous'` it so could simply be removed.

Reply:
Thanks @mike-000, I removed those.